recipe/meta.yaml

```diff
@@ -40,7 +40,7 @@ requirements:
     - pandas
     - pillow
     - protobuf 3.5.*
-    - python
+    - python  {{ pin_python }}
```

Comment:
This should not be needed if python is on `build` and `run` things should be properly pinned to python already.

Reply:
I agree. But the problem is when I try to create an environment like
```
conda create -n foo-caffe caffe
```
or if a try if I try to just install caffe, conda will hang forever. (I also opened a bug on conda about it)
An workaround is to pin python and ask conda to install caffe such as

```
conda create -n foo-caffe-pin python=3.7 caffe
```
it will just take few seconds

Reply:
The problem there is the channel mix-slowness. `conda-forge` has caffe for py27 only and the solver is having a hard time balancing conda-forge deps with defaults to get you caffe for py37. In that case you are better off with an env with only defaults.

PS: #49 should solve that.

Reply:
got it!
Thanks for the explanation! :)